http/src/request/application/command/create_guild_command/chat_input.rs

```diff
@@ -44,6 +45,7 @@ impl<'a> CreateGuildChatInputCommand<'a> {
         name: &'a str,
         description: &'a str,
     ) -> Result<Self, CommandValidationError> {
+        validate_name(name)?;
```

Comment:
i think we should have a newline after this, and also call the functions in alphabetical order